### PR TITLE
WIP: admin-guide - chapter 8 - process

### PR DESCRIPTION
**Pull request by alemorvan**
## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [ ] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by alemorvan** [comment]
@sspencerwire Next chapter is ready

File: en/rocky/8/admin_guide/08-process.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+Each time a file is accessed, the system checks the rights of the process according to its effective identifiers.
+
+## Process management
+
+A process cannot be run indefinitely, as this would be to the detriment of other running processes and would prevent multitasking.
+
+The total processing time available is therefore divided into small ranges and each process (with a priority) accesses the processor in a sequenced manner. The process will take several states during its life among the states:
+
+* ready: waiting for the availability of the process ;
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:148]: ... process;

File: en/rocky/8/admin_guide/08-process.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+* in execution: accesses the processor;
+* suspended: waiting for an I/O (input/output);
+* stopped: waiting for a signal from another process;
+* zombie: request for destruction;
+* dead : the father of the process kills his son.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:153]: dead: ...

File: en/rocky/8/admin_guide/08-process.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+2. Release of the used memory;
+3. Sending a signal to the parent and child processes.
+
+When a parent process dies, its children are said to be orphans. They are then adopted by the `init` process which will destroy them.
+
+### The priority of a process
+
+The processor works in time sharing, each process occupies a quantum of processor time.
+
+The processes are classified by priority whose value varies from **-20** (the highest priority) to **+20** (the lowest priority).
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:167]: ... to **+19** (the lowest priority).

File: en/rocky/8/admin_guide/08-process.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+```
+$ kill -9 1664
+```
+| Code | Signal    | Description                                            |
+|------|-----------|--------------------------------------------------------|
+| `2`  | _SIGINT_  | Immediate termination of the process                   |
+| `9`  | _SIGKILL_ | Interrupt the process (<kbd>CTRL</kdb> + <kdb>D</kdb>) |
+| `15` | _SIGTERM_ | Clean termination of the process                       |
+| `18` | _SIGCONT_ | Resume the process                                     |
+| `19` | _SIGSTOP_ | Suspending the process                                 |
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:204]: Suspend the process

File: en/rocky/8/admin_guide/08-process.md [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+
+> :star: **TIP:**
+The complete list of signals taken into account by the `kill` command is available by typing the command :
+```
+$ man 7 signal
+```
+
+### `nohup` command
+
+`nohup` allows to launch a process independent of a connection.
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:216]: independently

File: en/rocky/8/admin_guide/08-process.md [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+
+The `top` command displays the processes and their resource consumption.
+
+```
+$ top
+PID  USER PR NI ... %CPU %MEM  TIME+    COMMAND
+2514 root 20 0       15    5.5 0:01.14   top
+```
+
+| Colonne   | Description           |
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:344]: Column

**Review by SergeCroise — CHANGES_REQUESTED** [review]

File: en/rocky/8/admin_guide/08-process.md [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+
+The result is displayed in columns:
+
+```
+# ps -ef
+UID  PID PPID C STIME  TTY TIME      CMD
+root 1   0    0 Jan01  ?   00:00/03  /sbin/init
+```
+
+| Colonne  |  Description                |
```
Comment (SergeCroise) [inline en/rocky/8/admin_guide/08-process.md:90]: Column

**Review by SergeCroise — CHANGES_REQUESTED** [review]

**Comment by sspencerwire** [comment]
Will be merging into rl_administrator_docs, and then adding in SergeCroise editing suggestions before doing my own editing passes.

**Pull request merged**